spec/models/comment_spec.rb

```diff
@@ -1,5 +1,38 @@
 require 'rails_helper'
 
 RSpec.describe Comment, type: :model do
```

Comment:
Please add unit test for `#update_comments_counter` of Comment model and `update_likes_counter` of Like model.
It's a requirement to test all custom methods in the models.